gradle-baseline-java/src/main/groovy/com/palantir/baseline/BaselineFindBugsExtension.groovy

```diff
@@ -0,0 +1,11 @@
+package com.palantir.baseline
+
+import java.util.regex.Pattern
+
+class BaselineFindBugsExtension {
```

Comment:
can we add directly to the findbugs extension? not saying we should, just curious if that's even an option...